src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_rsa.c

```diff
@@ -46,6 +46,8 @@ static int GetOpenSslPadding(RsaPadding padding)
         case NoPadding:
             return RSA_NO_PADDING;
     }
+
+    return RSA_NO_PADDING;
```

Comment:
gcc analyzer seems to ignore the `assert`, and expects the non-void function to return something in all cases.

Reply:
I guess the reason is that assert is debug build only, it does nothing in release builds, so the analyzer is correct.